timezone_converter/main.py

```diff
@@ -86,6 +94,10 @@ def main() -> int:
             args.zone,
             args.hour,
         ).print_table()
+    elif args.search:
+        returncode = SearchView(
+            args.search,
+        ).print_search_results()
```

Comment:
This should be placed before the `args.timezone` check to avoid unwanted behaviors.

Also, the call to the method should fit in a single line (unless `Black` decides it doesn't...) so it would look like this:

```py
    if args.list:
        returncode = ListView(args.list).print_columns()
    elif args.search:
        returncode = SearchView(args.search).print_search_results()
    elif args.timezone:
		...
```